index.html

```diff
@@ -9963,8 +9999,10 @@ <h4 id="Beast Shape - Large Animal">Beast Shape - Large Animal</h4>
                </td>
                <td>Take Back to Den
                </td>
-               <td>Bite onto an enemy within 5ft, then move and drag the enemy up to 20ft
-                  with you. 2 FORT to resist
+               <td>Bite onto an enemy within 5ft that is <a class="condition" data-tooltip-container>stunned</a>, <a class="condition" data-tooltip-container>prone</a>,
+                  <a class="condition" data-tooltip-container>immobilised</a>, <a class="condition" data-tooltip-container>unconscious</a>, or 
+                  <a class="condition" data-tooltip-container>paralysed</a> then move and drag the enemy up to 20ft
+                  with you. 2 FORT to resist.
```

Comment:
Not sure about this needing a prerequisite? Is that because its meant to be infinite use? If so it's still too goodcause of infinite source of resistance drain/also too situational.